### ISSUE TITLE
removeChild not accessible due to internal protection

### ISSUE THREAD

**Issue by timroesner** [applidium/ADOverlayContainer#26]
I saw that in version 1.3 you added the extension snippet to the read me setup. In version 1.1.1 this was previously accessible and used by us. Any particular reason why this extension is not exposed anymore? https://github.com/applidium/ADOverlayContainer/blob/c8d65bcb0216af1a117b4cce6af5218b84e9c263/Source/Classes/Internal/Extensions/UIViewController%2BChildren.swift

**Comment by gaetanzanella** [comment]
Hi @timroesner, thanks for the feedback.

As I explained [here](https://github.com/applidium/ADOverlayContainer/commit/c8d65bcb0216af1a117b4cce6af5218b84e9c263#diff-1412df892081e7060e39d200f596cb95), I expected most of the clients to alright have their own extension to easily manage `UIViewController` children. Such an extension could be used everywhere and would conflict with the library's extension. Since the beginning, the extension was exposed by mistake.

Nevertheless, I made a mistake. I incremented the minor version whereas the change is not backwards-compatible.
